schema/preset.schema.yaml

```diff
@@ -155,6 +164,15 @@ properties:
             A list of utilities for handling fields, including managing unreferenced fields and extracting fields
             from hierarchical or JSON objects.
           $ref: '#/components/schemas/FieldUtils'
+required:
+  - name
+  - author
+  - description
+  - title
+  - iconURL
+  - autoloader
+  - silver
+  - gold
```

Comment:
I'm not sure why autoloader, silver and gold are required in the preset.  Is this a mistake or was there a specific ask for this perhaps?